drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/factory/remote/reusable/TestRemoteWebDriverFactorySessionStoring.java

```diff
@@ -107,6 +108,12 @@ public void setupMocks() {
 
         initializationParameter = new InitializationParameter(hubUrl, desiredCapabilities);
 
+        String browser = System.getProperty("browser", "phantomjs").toLowerCase();
```

Comment:
Why do you use the default value? The retrieved value is used only for checking if `chromeheadless` is used - then you don't care about some defaults....